listenbrainz/webserver/static/js/src/Scrobble.ts

```diff
@@ -91,7 +91,7 @@ export default class Scrobble {
         release_name: this.releaseName(),
         additional_info: {
           listening_from: "lastfm",
-          recording_mbid: this.trackMBID(),
+          lastfm_recording_mbid: this.trackMBID(),
```

Comment:
Maybe we should rename this field as `lastfm_track_mbid`?

Reply:
Makes sense.